server/conf_test.go

```diff
@@ -200,15 +200,21 @@ func TestParseConfig(t *testing.T) {
 	if !opts.Clustering.Bootstrap {
 		t.Fatal("Expected Bootstrap to be true, got false")
 	}
-	peers := []string{"b", "c"}
-	if len(peers) != len(opts.Clustering.Peers) {
+	peers := []string{"a", "b", "c"}
```

Comment:
I would revert the changes to this test. This test parses a configuration file (`test/configs/test_parse.conf`) and check that the loaded options matches what is in the file.
If you want to add a test, it would be at the end of the test `TestParseConfigureOptions`. At the end of this test, you see a case where we check node_id and peers. You could add a similar one with `a` being present on node_id and peers and check that it is then not in the array. I can add the test after merging the PR if you prefer.